internal/saucecloud/cloud.go

```diff
@@ -489,6 +489,9 @@ func (r *CloudRunner) archiveNodeModules(tempDir string, rootDir string, matcher
 	// does the user only want a subset of dependencies?
 	if hasMods && wantMods {
 		reqs := node.Requirements(filepath.Join(rootDir, "node_modules"), r.NPMDependencies...)
+		if len(reqs) == 0 {
+			return "", fmt.Errorf("unable to find required dependencies. Please check 'node_modules' folder and make sure the dependencies exist")
```

Comment:
Just a bit nicer formatting due to sentence restrictions in error messages

```suggestion
			return "", fmt.Errorf("unable to find required dependencies; please check 'node_modules' folder and make sure the dependencies exist")
```